docs/libcurl/opts/CURLOPT_SSL_OPTIONS.3

```diff
@@ -57,8 +57,8 @@ library). If combined with \fICURLSSLOPT_NO_REVOKE\fP, the latter takes
 precedence. (Added in 7.70.0)
 .IP CURLSSLOPT_NATIVE_CA
 Tell libcurl to use the operating system's native CA store for certificate
-verifiction. Works only on Windows when built to use OpenSSL. This option
-overrides \fICURLOPT_CAINFO(3)\fP if both are set. (Added in 7.71.0)
+verification. Works only on Windows when built to use OpenSSL.
+(Added in 7.71.0)
```

Comment:
Maybe this explanation could add that this verification is *in addition* to the possible verification done using CAINFO/CAPATH ?

Reply:
I think that's a good idea but I consider these temporary changes for the impending release, so for that reason I didn't document it that way. This issue can stay open to gather feedback, and possibly it is not the right way to go. If it goes in the documentation we don't have any wiggle room.

Reply:
The problem is then that nobody can really reliably use this feature as it isn't documented how exactly it is expected to work when combined with the existing verify option, so some people might instead assume something...

Reply:
How about I add "This option is experimental." or "This option is in beta." or something like that

Reply:
That would work - and if so, add it to `docs/EXPERIMENTAL.md`